core/src/test/java/jenkins/model/NewViewLinkTest.java

```diff
@@ -27,7 +27,6 @@ public void getActionsHasPermission() throws Exception {
         final Action action = actions.get(0);
         assertEquals(Messages.NewViewLink_NewView(), action.getDisplayName());
         assertEquals(NewViewLink.ICON_FILE_NAME, action.getIconFileName());
-        assertEquals(NewViewLink.URL_NAME, action.getUrlName());
```

Comment:
I would rather rewrite the assert to check the full URL or to chack whether the URL contains the name. Just removing the check is a bit hackish